### PR TITLE
Update session manager changes, auth-service v9 updates and wallet-services v3 updates

### PR DESCRIPTION
**Pull request by grvgoel81**
## Motivation and Context
* Update session-manager-android sdk.
* Update authService version to v9 and walletServices to v3.

Jira Link: https://toruslabs.atlassian.net/browse/PD-3861

## Description
* Update session-manager-android sdk and related changes wrt API method parameters.
* Update authService version to v9 and walletServices to v3.

## How has this been tested?
* Tested all functionalities like login, logout, session management ,walletservices, setupMFA and request() method.

## Screenshots (if appropriate):

https://github.com/user-attachments/assets/81692e5b-327a-4445-82d9-6803104fd2c0

For request method changes:

https://github.com/user-attachments/assets/22677424-f357-4bd9-9634-b45dbfff976a

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project. (run lint)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by metalurgical** [comment]
Merge #72 into this.

**Review by Sopheap-cloud — APPROVED** [review]

**Comment by metalurgical** [comment]
Note #74 must be merged before this PR, then this PR must be rebased on top of it.

File: Sources/Web3Auth/Web3Auth.swift [hunk 1]
```diff
@@ -273,80 +280,88 @@ public class Web3Auth: NSObject {
...
             })
-        }
-        else {
+        } else {
             throw Web3AuthError.runtimeError("SessionId not found. Please login first.")
         }
     }
-    
+
     public func launchWalletServices(chainConfig: ChainConfig, path: String? = "wallet") async throws {
```
Comment (chaitanyapotti) [inline Sources/Web3Auth/Web3Auth.swift:312]: take input of appState as an optional field

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**